modules/core/src/main/java/org/apache/ignite/internal/managers/encryption/GridEncryptionManager.java

```diff
@@ -311,6 +311,8 @@ public GridEncryptionManager(GridKernalContext ctx) {
             }
         });
 
+        ctx.internalSubscriptionProcessor().registerGlobalStateListener(this);
```

Comment:
`GridEncryptionManager` doesn't call `onKernalStart` on activation. As for `GridTaskProcessor`,  investigation is required and this activity is out of scope of this ticket.